zone/CMakeLists.txt

```diff
@@ -548,6 +548,7 @@ SET(gm_commands
     gm_commands/unmemspells.cpp
     gm_commands/unscribespell.cpp
     gm_commands/unscribespells.cpp
+	gm_commands/updatechecksum.cpp
```

Comment:
Formatting, this file is weird, think it's spaces.

Reply:
Fixed.